src/js/entities-service/entities/roles.js

```diff
@@ -0,0 +1,22 @@
+import BaseEntity from 'js/base/entity-service';
+import { _Model, Model, Collection } from './entities/roles';
+
+const Entity = BaseEntity.extend({
+  Entity: { _Model, Model, Collection },
+  radioRequests: {
+    'roles:model': 'getModel',
+    'roles:collection': 'getCollection',
+    'fetch:role:model': 'fetchRole',
```

Comment:
I think we can remove `fetchRole` entirely.  Until a use case arises, roles should be treated like `states` Just options available to the organization.